gammapy/cube/models.py

```diff
@@ -437,3 +438,49 @@ def compute_npred(self):
         if self.background:
             npred.data += self.background.data
         return npred.data
+
+
+class SkyMap3d(object):
+    """Cube sky map template model.
+
+    Parameters
+    ----------
+    map : `~gammapy.map.Map`
+        Map template
+    norm : `~astropy.units.Quantity`
+        Norm parameter (multiplied with map values)
```

Comment:
Should give a flux unit.
Is the value expected to be integral of differential in a given energy bin? Maybe add a flag to check this no?

Reply:
I think `norm` should be a float (change that in the docstring), a unitless multiplicative normalisation factor. The diffuse model map cubes should already come with a unit and the appropriate thing (diffuse differential surface brightness e.g. in `cm-2 s-1 MeV-1 sr-1`).

Reply:
It's questionable if we want to have this `norm` parameter at all.
It would be worth a look how in Fermi-LAT analysis people change the diffuse model by a constant or even power-law spectrum, and how they handle that model composition and units.
We should probably do the same.

But for now, just this norm=1 no-op should give you what you need for DC1, and you should even be able to fit that norm parameter and it should come out ~ 1 .